index.js

```diff
@@ -92,8 +97,8 @@ module.exports = function (config) {
                 if (i == 0) {
                     tests = '"tests" : [';
                     // if this is the first test that is added to the list of tests executed.
-                    tests = tests + "" + '{"testKey":"' + test.tags[1].split("@")[1] + '","status":"' + test.state + '", "examples":' + JSON.stringify(test.examples) + ',"comment" : "' + test.comment + '" }';
-                } else tests = tests + "" + ',{"testKey":"' + test.tags[1].split("@")[1] + '","status":"' + test.state + '", "examples":' + JSON.stringify(test.examples) + ',"comment" : "' + test.comment + '" }';
+                    tests = tests + "" + '{"testKey":"' + test.tags[1].split("@TEST_")[1] + '","status":"' + test.state + '", "examples":' + JSON.stringify(test.examples) + ',"comment" : "' + test.comment + '" }';
```

Comment:
this is not ok of server version.
the tag, can be '@ZZZ-1122' or something else.
If it's always like '@TEST_'  on cloud version, you can also add an if cloud